testing/sample_framework/src/android/android_app_framework.cc

```diff
@@ -23,6 +23,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <ctime>
+#include <iostream>
```

Comment:
Do you still need `#include <iostream>`?

Reply:
Deleted.